### PR TITLE
LPD-35323 Multi Shipping tab in Order Details

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-35323

This pr just has the multishipping tab added onto the order details page. The tab is currently empty. I also included a playwright test to make sure that the tab is only visible when the multishipping configuration is toggled and the site type is b2b.

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/taglib/CommerceOrderMultishippingScreenNavigationEntry.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+					_configurationProvider.getConfiguration(
+						CommerceAccountGroupServiceConfiguration.class,
+						new GroupServiceSettingsLocator(
+							commerceChannel.getGroupId(),
+							CommerceConstants.SERVICE_NAME_COMMERCE_ACCOUNT));
+
+			int commerceSiteType =
+				commerceAccountGroupServiceConfiguration.commerceSiteType();
+
+			if (commerceSiteType == CommerceChannelConstants.SITE_TYPE_B2B) {
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/taglib/CommerceOrderMultishippingScreenNavigationEntry.java:67]: not sure we have to restrict this to B2B channels, I'll keep you posted on that

**Review by luca-pellizzon — COMMENTED** [review]
I'll check with Giulia and Gianmarco, but I think this development is only for "new order pages" not to integrate on the old layouts.

**Comment by dsitu** [comment]
Closing as this development will only be done in the new order pages fragments/site-initializer

**Pull request closed without merging**